src/lib/apiTypesV2/Entity.cpp

```diff
@@ -81,27 +81,39 @@ std::string Entity::render(ConnectionInfo* ciP, RequestType requestType, bool co
 
   if ((errorCode.description == "") && ((errorCode.error == "OK") || (errorCode.error == "")))
   {
-    std::string out = "{";
-
-    out += JSON_VALUE("id", id);
-    out += ",";
-    out += JSON_STR("type") + ":" + ((type != "")? JSON_STR(type) : "null");
-
-    if (attributeVector.size() != 0)
+    if (renderMode == "values")
     {
+      std::string out = "[";
+      if (attributeVector.size() != 0)
+      {
+        out += attributeVector.toJson(true, false, renderMode, ciP->uriParam["attrs"]);
+      }
+      out += "]";
+      return out;
+    }
+    else
```

Comment:
As you return after 'renderMode == "values"', the else part can be removed, I mean, the content of the else part can be outside the if-else and be less indented.